client/js/bundles/user/unregisteredPublicagencyList.js

```diff
@@ -10,9 +10,9 @@
 /**
  * This is the entry point for unregistered_publicagency_list.html.twig
  */
-import { DpToggleForm } from '@demos-europe/demosplan-ui'
 import { initialize } from '@DpJs/InitVue'
+import ToggleForm from '@DpJs/components/user/DpToggleForm'
```

Comment:
Lets also rename the file, then!

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2026/commits/d306a4b4b0003e10d8171889584c790c8ce14932
Thanks!